src/bentoml/__init__.py

```diff
@@ -29,7 +29,7 @@
 
 # BentoML built-in types
 from ._internal.bento import Bento
-from ._internal.cloud import YataiClient
```

Comment:
hmm what about OSS yatai users?

Reply:
good question, in fact the yatai client has been broken for a long time. you can't run it on main nor recent releases. i think we intentionally don't put effort in it. so i didn't touch it. everything remains as it is for yatai

Reply:
fair enough.